plugins/chat/assets/javascripts/discourse/services/chat-history.js

```diff
@@ -0,0 +1,22 @@
+import Service from "@ember/service";
+import { tracked } from "@glimmer/tracking";
+
+export default class ChatHistory extends Service {
+  @tracked history;
+
+  get previousRoute() {
+    if (this.history?.length > 1) {
```

Comment:
Shouldn't this return `this.history[0]` if this is false? Otherwise we could have 1 previous route and it returns null/undefined.

Reply:
I might be missing something but to me history.length == 1 means we only have current route, we have a previous route once we start to have two entries.

Reply:
Ohhh right yeah disregard :+1: